pkgs/tools/audio/yabridgectl/libyabridge-from-nix-profiles.patch

```diff
@@ -0,0 +1,68 @@
+diff --git a/tools/yabridgectl/src/config.rs b/tools/yabridgectl/src/config.rs
+index fbb6b97..1f48797 100644
+--- a/tools/yabridgectl/src/config.rs
++++ b/tools/yabridgectl/src/config.rs
```

Comment:
Did you get this from a PR or commit? Please use fetchpatch to get this if so. Otherwise I would prefer upstreaming it if possible.

Reply:
No, I wrote this myself. It's a patch specific to Nix, so I don't think something like this would be accepted upstream.

Reply:
Normally it is not great to maintain patches in nixpkgs because it requires more maintenance work.

Reply:
Yeah :confused:, but without the patch `yabridgectl` won't be able to find `libyabridge.so`. I'm okay with maintaining this patch, I have a pretty good workflow setup for it.

I'll try to see if I can refactor it into something that can be upstreamed, but it would be more complicated than this patch, so it would probably be better to keep this for now.